packages/flutter/lib/src/material/bottom_sheet.dart

```diff
@@ -288,7 +318,16 @@ class _BottomSheetState extends State<BottomSheet> {
       clipBehavior: clipBehavior,
       child: NotificationListener<DraggableScrollableNotification>(
         onNotification: extentChanged,
-        child: widget.builder(context),
+        child: (widget.enableDrag && hasDragHandle) ? Stack(
```

Comment:
This would be easier to read if the simple `widget.builder(context)` was first, or just created via a short-circuit `if`:
```dart
if (!hasDragHandle || !widget.enableDrag) {
  return widget.builder(context);
}

return Stack(
   ...
);
```